libr/core/cmd_open.c

```diff
@@ -1427,6 +1427,8 @@ static int cmd_open(void *data, const char *input) {
 		}
 		{
 			const char *argv0 = argv ? argv[0] : ptr;
+			// Unescape spaces from the path
+			r_str_replace ((char *)argv0, "\\ ", " ", true);
```

Comment:
this (char*) cast is terribly scary

Reply:
Unfortunately r_str_replace works on the given string, ptr is const and r_core_file_open expects const. I could make a new function just for r_str_path_unescape but is it really scary at all?

Reply:
It doesn’t looks safe. It may return in a different heap and cause an UAF